app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -371,8 +396,8 @@ private void onShraeClicked(final BrowserFragment browserFragment) {
     @Override
     public View onCreateView(String name, Context context, AttributeSet attrs) {
         if (name.equals(IWebView.class.getName())) {
-            View v = WebViewProvider.create(this, attrs);
-            return v;
+            mWebkitView = (WebkitView) WebViewProvider.create(this, attrs);
```

Comment:
Don't keep Webview here, it's would prevent the Fragment from being recycled. Try browserFragment.reload(); (See onRefreshClicked and on how we get The browserFragment)